web/src/components/PasswordWithMeter.js

```diff
@@ -107,7 +114,9 @@ Password.defaultProps = {
   onChange: () => {},
   title: 'Password',
   value: '',
-  showMeter: false
+  showMeter: false,
+  errorMessage: '',
+  customErrorMessage: ''
 };
 
 export default Password;
```

Comment:
do we still need this file? Isn't it used to create a password?

Reply:
It's the password field with or without the meter so we still need it I believe. It could be refactored to be `Password.js` and the meter and unnecessary bits could be removed.